packages/storage/src/components/UpdateUserPhotoModal/index.js

```diff
@@ -0,0 +1,194 @@
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { Button, RenderIf, Spinner } from 'react-rainbow-components';
+import { useFirebaseApp, useCurrentUserState } from '@rainbow-modules/firebase-hooks';
+import { nanoid } from 'nanoid';
+import {
+    getStorage,
+    ref,
+    getDownloadURL,
+    uploadBytesResumable,
+    deleteObject,
+} from 'firebase/storage';
+import { updateProfile } from 'firebase/auth';
+import EditRemoveDialog from './editRemoveDialog';
+import {
+    StyledModalEdit,
+    StyledModalFooter,
+    StyledButtonFocus,
+    StyledCropperContainer,
+    StyledCropper,
+    StyledModal,
+    Loading,
+    StyledIcon,
+} from './styled';
+import RotateLeftIcon from './rotateLeftIcon';
+
+export default function UpdateUserPhotoModal(props) {
+    const { path, photo, bucket, isOpen, onRequestClose, avatarInitials, handlePhotoUrl } = props;
+
+    const app = useFirebaseApp();
+    const { user, reload } = useCurrentUserState() || {};
+
+    const pathProp = path || `users/${user ? user.uid : ''}/profilePhoto`;
+    const bucketProp = bucket || `${app.options.storageBucket}`;
+
+    const [photoURL, setPhotoURL] = useState(photo);
+    const [isOpenEdit, setIsOpenEdit] = useState(false);
+    const cropperRef = useRef(null);
+    const [loading, setLoading] = useState(false);
```

Comment:
normally for booleans we have a convention to use the `is` word as prefix in this case you should use `isLoading` this is for indicate that it is a boolean